server.js

```diff
@@ -9,6 +10,7 @@ app.use('/', routes);
 
 // Run application on configured port
 if (config.env === 'development') {
+  dotenv.config();
```

Comment:
do we need this? we don't have a .env file

Reply:
It is probably because I have this in the config.js
`  apiHost: process.env.API_HOST || 'http://localhost:8080/api/v1',`
so it would default to localhost if it wasn't set

Reply:
my `.env` currently just has this
`API_HOST="http://localhost:8080/api/v1"`

Reply:
yeah so probably don't need that package as you check if config.env is development then use the defaults in the config file. If we keep the defaults in the config file and .env there will be 2 sources of truth

Reply:
true don't really need that .env file then. I'll remove it. Probably means we don't really need the dotenv at all either and that line you just mentioned